### PR TITLE
Iterate through pages in VSAC API

### PR DESCRIPTION
**Pull request by dogversioning**
This PR makes the VSAC API support the pagination paramter for the expand operation.

### Checklist
- [X] Consider if documentation (like in `docs/`) needs to be updated
- [X] Consider if tests should be added
- [X] Update template repo if there are changes to study configuration

### CODE REVIEW

File: cumulus_library/apis/umls.py [hunk 1]
```diff
@@ -82,7 +86,7 @@ def get_vsac_valuesets(
         if oid:
             url = f"{url}/{oid}"
         # TODO: Do we need to support the remaining FHIR operators?
-        if action == "expansion":
+        if action == "expansion" and "expand" not in url:
```
Comment (mikix) [inline cumulus_library/apis/umls.py:89]: nit: `"/$expand" not in url`?

File: cumulus_library/apis/umls.py [hunk 2]
```diff
@@ -91,16 +95,32 @@ def get_vsac_valuesets(
...
         if response.status_code == 404:
             raise errors.ApiError(f"Url not found: {url}")
         all_responses = [response.json()]
+        if action == "expansion":
+            # Do we need to fetch the next page?
+            # Note: the VSAC API hard codes the page size to 1000
+            # and ignores the count param in the FHIR API
+            # (https://www.hl7.org/fhir/ValueSet-operation-expand.html),
+            # so we just hardcode an expansion of 1000 and hope it
+            # doesn't chage
```
Comment (mikix) [inline cumulus_library/apis/umls.py:112]: cool

File: cumulus_library/apis/umls.py [hunk 3]
```diff
@@ -91,16 +95,32 @@ def get_vsac_valuesets(
...
+            if all_responses[0]["expansion"]["total"] > offset + 1000:
```
Comment (mikix) [inline cumulus_library/apis/umls.py:113]: That `total` value is consistent as you walk through the results? i.e. the second response's total value isn't like "total left to go" or anything?

**Review by mikix — APPROVED** [review]

File: cumulus_library/apis/umls.py (hunk 3 above)
Comment (dogversioning) [reply cumulus_library/apis/umls.py:113]: correct, it's always the dataset full size.

**Pull request merged**